schema/schema.sql

```diff
@@ -0,0 +1,84 @@
+DROP TABLE IF EXISTS track_tag_assignment;
+DROP TABLE IF EXISTS album_tag_assignment;
+DROP TABLE IF EXISTS tag;
+DROP TABLE IF EXISTS track_purchase;
+DROP TABLE IF EXISTS album_purchase;
+DROP TABLE IF EXISTS country;
+DROP TABLE IF EXISTS track;
+DROP TABLE IF EXISTS album;
+DROP TABLE IF EXISTS artist;
+DROP TABLE IF EXISTS subscriber;
```

Comment:
Do `DROP TABLE IF EXISTS track_tag_assignment, album_tag_assignment, tag, track_purchase.....etc` so that the order of dropping doesn't matter

Reply:
I prefer having it in this way because it helps me keep track of the heirarchy of tables. Thank you for the suggestion though.